packages/react-dom/server.browser.js

```diff
@@ -9,4 +9,10 @@
 
 'use strict';
 
-module.exports = require('./src/server/ReactDOMServerBrowser');
+var ReactDOMServer = require('./src/server/ReactDOMServerBrowser');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest and handles https://github.com/facebook/react/issues/11526
```

Comment:
I would split the new verbiage into a separate line.

```js
// Note: when changing this, also consider <link>
```

Reply:
sure thing. updated